Add release notes for OEE 10.17 (off cycle). (#281)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/whatsnew-10-17-0.md

In the **User** page, at the right of the top menu bar, click **Revoke tokens** to revoke all session tokens immediately.

![User token revocation](/images/release-notes/admin-users-list.png)

### OEE App

An updated version of the Cumulocity IoT OEE application is now available. 

> This version adds support for defining one-off and repeating work Shifts on a per-location basis. Doing so will allow calculation of OEE (Overall Equipment Efficiency), Availability, Performance, Quality and various other metrics over these timeframes. 

Reviewer: ```suggestion
This version adds support for defining one-off and repeating work shifts on a per-location basis. This allows the calculation of OEE (Overall Equipment Efficiency), Availability, Performance, Quality and various other metrics over these timeframes. 
```

![Admin shift plan configuration](/images/release-notes/oee-admin-shift-plans.png)

> For details, refer to the OEE Guide.

Reviewer: Add link

Author: Done, although I couldn't find a suitable shortcode for the release as per previous URLs. I'm sure there probably is one, let me know if I missed it.

> For details, refer to the [Cumulocity IoT OEE](https://cumulocity.com/guides/10.17.0/oee/overview/) section of the *User guide*

Reviewer: ```suggestion
For details, refer to the [Cumulocity IoT OEE guide](https://cumulocity.com/guides/10.17.0/oee/overview/).
```
